.bazelrc

```diff
@@ -44,7 +49,27 @@ build --cxxopt='-Wno-final-dtor-non-final-class'
 build --host_cxxopt='-Wno-final-dtor-non-final-class'
 build --cxxopt='-Wno-unused-const-variable'
 build --host_cxxopt='-Wno-unused-const-variable'
+build --cxxopt='-Wno-unused-variable'
+build --host_cxxopt='-Wno-unused-variable'
+build --cxxopt='-Wno-unused-function'
+build --host_cxxopt='-Wno-unused-function'
```

Comment:
Are these for ZetaSQL as well? If so, move it after the comment on line 56?

Reply:
There are two sections here: warnings we are turning off for ZetaSQL for which I have proposed fixes commits to the ZetaSQL folks that should address the warnings, and warnings that we are turning off for ZetaSQL where I don't know if a fix will be published soon. These are in the former category, and there's already a header addressing them.